HousingFinanceInterimApi/V1/UseCase/GenerateReportUseCase.cs

```diff
@@ -41,13 +37,18 @@ public class GenerateReportUseCase : IGenerateReportUseCase
             IReportGateway reportGateway,
             ITransactionGateway transactionGateway,
             IGoogleFileSettingGateway googleFileSettingGateway,
-            IGoogleClientService googleClientService)
+            IGoogleClientService googleClientService,
+            int sleepDuration = 30_000, // ms
+            int retryInterval = 200 // ms
```

Comment:
You didn't like 1s? :joy: 

200ms is a bit cutting it close when it comes to network requests I think.
Hopefully we won't get rate limit errors :D

Reply:
Actually, thinking this over... maybe it would be cleaner to have an options object to wrap around these non-dependency options like is often the convention in C#?

Reply:
Reverted to 1s, didn't intend to modify prod functionality :+1: 
https://github.com/LBHackney-IT/housing-finance-interim-api/pull/150/commits/24e3072a0c14499fba8101ff1026722102c6907a